src/org/thoughtcrime/securesms/database/DatabaseFactory.java

```diff
@@ -1426,6 +1436,38 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
         db.execSQL("ALTER TABLE recipient_preferences ADD COLUMN nick_name TEXT DEFAULT NULL;");
       }
 
+      if(oldVersion < INTRODUCED_MESSAGE_REACTION) {
+        db.execSQL(MessageReactionDatabase.CREATE_TABLE);
+        db.execSQL("ALTER TABLE sms ADD COLUMN hash TEXT;");
+        db.execSQL("ALTER TABLE mms ADD COLUMN hash TEXT;");
+
+        Cursor smsCursor = db.rawQuery("SELECT _id, address, date_sent FROM sms", null);
```

Comment:
Shouldn't the static variables be used here instead of hardcoding it?

Reply:
should also be used in the following lines in that case.